src/core/render.js

```diff
@@ -0,0 +1,41 @@
+//Include helper functions and Popper
+import Popper from 'popper.js';
```

Comment:
Should this use require or does it need to be imported?
```js
const Popper = require('popper.js')
```

Reply:
Not sure if there is a way to get to work with require. But it seems the slight differences between require  and import cause an error to be thrown. 

This throws a type error 
```
var Popper = require('popper.js');
var p = new Popper(refObject, target, options) 
```

This succeeds. 
```
import Popper from 'popper.js';
var p = new Popper(refObject, target, options);
```

Reply:
We should be using `require()` as a rule until `import` is well supported in Node.  Node 8 / npm 5 is still a bit iffy (behind a flag and only using the `.mjs` extension).  Maybe in a year or two, `import` will be practical for new projects.

`const Popper = require('popper.js')` seems to work for me on node@6.  I would expect webpack to be the same.  You could always `console.log(Popper)` to see what you've got.